src/Functions/printToLog.cpp

```diff
@@ -0,0 +1,74 @@
+#include <DataTypes/DataTypeString.h>
+#include <Columns/ColumnString.h>
+#include <Columns/ColumnConst.h>
+#include <Columns/ColumnsNumber.h>
+#include <Functions/FunctionFactory.h>
+#include <Functions/FunctionHelpers.h>
+#include <IO/WriteHelpers.h>
+#include <IO/WriteBufferFromOStream.h>
+
+#include <common/logger_useful.h>
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int ILLEGAL_TYPE_OF_ARGUMENT;
+}
+
+class FunctionPrintToLog : public IFunction
+{
+public:
+    static constexpr auto name = "printToLog";
```

Comment:
Can I now run `select printToLog(repeat('a', 1000)) from system.numbers` to bring down the playground? It should require sysadmin priveleges and be disabled in read only mode.

Reply:
`allow_introspection_functions`

Reply:
I also wanted this function, see https://github.com/ClickHouse/ClickHouse/issues/9434

Details:
- only trace level;
- no quantity parameter (I think it's harmful);
- different name (log is also a verb);